tutor4points/core/templates/home.html

```diff
@@ -44,14 +45,21 @@ <h2>Tutor Requests</h2>
                                         {{tutor_request.tutor_date}} @<br>{{tutor_request.tutor_time}} <br/>
                                     </div>
                                     <div class = "respond-btns d-flex">
-                                        <button type="submit" name="accept-request" class="btn-secondary respond-button" 
-                                            data-toggle= "modal" data-target="#modal-accept-request">
-                                            Accept
-                                        </button>
-                                        <button type="submit" name="decline-request" class="btn-secondary respond-button" 
-                                        data-toggle= "modal" data-target="#modal-decline-request">
-                                            Decline
-                                        </button>
+                                        {% if tutor_request.accepted %}
```

Comment:
I don't see the "Complete" button appearing on the homepage for accepted requests on my end.